hyperwallet/tests/test_api.py

```diff
@@ -1393,6 +1393,119 @@ def test_list_paypal_accounts_success(self, mock_get):
 
         self.assertTrue(response[0].token, self.data.get('token'))
 
+    def test_create_paypal_account_status_transition_fail_need_user_token(self):
+
+        with self.assertRaises(HyperwalletException) as exc:
+            self.api.createPayPalAccountStatusTransition()
+
+        self.assertEqual(exc.exception.message, 'userToken is required')
+
+    def test_create_paypal_account_status_transition_fail_need_paypal_token(self):
+
+        with self.assertRaises(HyperwalletException) as exc:
+            self.api.createPayPalAccountStatusTransition('token')
+
+        self.assertEqual(exc.exception.message, 'payPalAccountToken is required')
+
+    def test_create_paypal_account_status_transition_fail_need_data(self):
+
+        with self.assertRaises(HyperwalletException) as exc:
+            self.api.createPayPalAccountStatusTransition('token', 'token')
+
+        self.assertEqual(exc.exception.message, 'data is required')
+
+    @mock.patch('hyperwallet.utils.ApiClient._makeRequest')
+    def test_create_paypal_account_status_transition_success(self, mock_post):
+
+        mock_post.return_value = self.data
+        response = self.api.createPayPalAccountStatusTransition('token', 'token', self.data)
+
+        self.assertTrue(response.token, self.data.get('token'))
+
+    def test_get_paypal_account_status_transition_fail_need_user_token(self):
+
+        with self.assertRaises(HyperwalletException) as exc:
+            self.api.getPayPalAccountStatusTransition()
+
+        self.assertEqual(exc.exception.message, 'userToken is required')
+
+    def test_get_paypal_account_status_transition_fail_need_check_token(self):
```

Comment:
Seems that "check" in the method's name should be replaced with "paypal_account". The same with some other methods in this file.

Reply:
Fixed.